### PR TITLE
feat: move extension name & description into locales

### PR DESCRIPTION
**Pull request by romaindequidt**
Refs: #287

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ This boilerplate is made for creating chrome extensions using React and Typescri
 ## Procedures: <a name="procedures"></a>
 
 1. Clone this repository.
-2. Change `name` and `description` in package.json => **Auto synchronize with manifest**
+2. Change `extensionDescription` and `extensionName` in messages.json
```
Comment (Jonghakseo) [inline README.md:61]: 👍🏼

File: manifest.js [hunk 2]
```diff
@@ -6,9 +6,10 @@ import packageJson from './package.json' assert { type: 'json' };
  */
 const manifest = {
   manifest_version: 3,
-  name: packageJson.name,
+  default_locale: 'en',
+  name: '__MSG_extensionName__',
```
Comment (Jonghakseo) [inline manifest.js:10]: How about adding the following comment to help users understand?

```suggestion
  /**
   * if you want to support multiple languages, you can use the following reference
   * https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization
   */
  name: '__MSG_extensionName__',
```

**Review by Jonghakseo — APPROVED** [review]
LGTM

File: manifest.js (hunk 2 above)
Comment (romaindequidt) [reply manifest.js:10]: Sorry for the delay @Jonghakseo & @PatrykKuniczak.
The change is pushed.

**Review by PatrykKuniczak — APPROVED** [review]
LGTM

**Comment by romaindequidt** [comment]
Please, it's work to rebase so many times. Can you merge please?

**Pull request merged**